samples/AudioPlayerSample/GlobalSuppressions.cs

```diff
@@ -0,0 +1,9 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>", Scope = "member", Target = "~F:AudioPlayerSample.ViewModels.AudioRecorderPageViewModel._RecordButtonText")]
```

Comment:
Instead of ignoring these can we fix them? 😊

Reply:
Oh yeah how did that creep in?